public/app/features/explore/ExplorePaneContainer.tsx

```diff
@@ -84,7 +83,6 @@ class ExplorePaneContainerUnconnected extends React.PureComponent<Props> {
 
   componentWillUnmount() {
     this.exploreEvents.removeAllListeners();
-    this.props.cleanupPaneAction({ exploreId: this.props.exploreId });
```

Comment:
This was the real cause of our problems. React will unmount children components when the parent rerenders. Hitting the split button would pass new parameters to SplitPaneWrapper, which means the ExplorePaneContainer would unmount and then be cleaned up. When the pane would then try to be used, it would not exist and everyone would be very sad about it. I solved this by moving the cleanup to the parent unmount in Wrapper, which will only clean up the panes if we navigate away from explore completely. I am not sure if there are side effects to this, but it seems like a good solution to me.